pkg/trace/obfuscate/sql_test.go

```diff
@@ -29,6 +30,14 @@ type sqlTokenizerTestCase struct {
 	expectedKind TokenKind
 }
 
+var sqlOnOptions = config.SQLObfuscationConfig{
```

Comment:
Can we please remove these variables? It is super easy to just type them as either:
* `config.SQLObfuscationConfig{}` for off
* `config.SQLObfuscationConfig{QuantizeSQLTables: true}` for on

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/d7e46cde49253718503b756a70f0254356f7d4e8